homeassistant/components/gios/config_flow.py

```diff
@@ -79,8 +78,7 @@ async def async_step_user(
                         sort=True,
                         mode=SelectSelectorMode.DROPDOWN,
                     ),
-                ),
-                vol.Optional(CONF_NAME, default=self.hass.config.location_name): str,
```

Comment:
Name was removed here, but not removed from the translations file (string.json), which still needs to be cleaned up.